platform/jewel/ui-tests/src/test/kotlin/org/jetbrains/jewel/ui/component/ListComboBoxUiTest.kt

```diff
@@ -491,6 +492,83 @@ class ListComboBoxUiTest {
         composeRule.onNodeWithTag("Book").assertIsSelected()
     }
 
+    @Test
+    fun `when editable ListComboBox text is edited then selectedIndex remains unchanged`() {
+        var selectedIdx = 1
```

Comment:
Can we spell out `index`? I had to stop and think "what does `idx` stand for?"

Reply:
Sorry. A bit of brain fog and a bit of “It's a-me AI. Me help!”.